extensions/amp-ad/0.1/test/test-amp-ad.js

```diff
@@ -107,7 +107,9 @@ function tests(name, installer) {
           type: 'a9',
           src: 'testsrc',
           resizable: '',
-        }, 'https://schema.org').then(element => {
+        }, 'https://schema.org', undefined, () => {
+          clock = sandbox.useFakeTimers();
```

Comment:
I think that is better. And we could pass a function reference in instead of an inline closure.